gaia-sdk-python/gaia_sdk/api/IdentityOp.py

```diff
@@ -29,5 +47,18 @@ def export(self, identity_id: str) -> Observable[bytes]:
             .pipe(
             ops.map(lambda response: response.content))
 
-    def import_identity(self, identity_id: str = None) -> None:  # 'import' keyword already taken
-        raise NotImplementedError("Implement identity import functionality is not yet implemented")
+    def import_identity(self, tenant_id: str, identity_name: str, content: bytes, override: bool = False,
+                        identity_id: str = None) -> Observable[IdentityImported]:
+        def complete_import(data_ref: DataRef) -> dict:
+            self._logger.debug(f"Started import of identity {identity_name}")
+            return self._client.post_json(IdentityImportImpulse(data_ref.uri, tenant_id, check_identity_id(identity_id),
+                                                                identity_name, override),
+                                          "/identity/import").json()
+
+        uri = "gaia://{}/identities/".format(tenant_id)
```

Comment:
Zur Info, man kann in Python noch gemütlicher Variablen in Strings einbauen wenn man ein f vor die Quotes schreibt.
some_var = f"value of tenant_id: {tenant_id}"